### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by DKSteffenDE** [Povstalec/StargateJourney#89] [General]
I have looked around, but haven't been able to find a roadmap, so was just wondering if you could share some of the plans for functionality down the line?
Maybe with what version they are planned for?

**Comment by Povstalec** [comment]
Yeah, roadmap isn't really something that's gonna work really. I just end up working on whatever I feel like I want to add next and while I do have things written down, there is no order to them.

I can only give a vague idea of what's to come (do keep in mind there is no actual order beyond the theme of the updates and even that doesn't really dicate what I add, since I already added a new planet in 0.6):

0.6 Technical changes
- Just generally adding technology and making the mod work properly on the technical side of things
- For specific stuff, I'm planning on adding more DHDs
- Dialing Computer (for people who don't want to use ComputerCraft to dial manually)
- Getting Transport Rings and the Ring Panel to a working state
- Energy generation (getting Naquadah Generators to a working state and adding some other ways of generating energy)

0.7 Adventure
- Abydos (terrain, biomes, structures and mobs)
- Jaffa
- Goa'uld
- Ra boss
- Chulak (better terrain like valleys and mountains, more interesting structures)
- Some more planets to explore

versions between 0.7 and 1.0, who knows what's gonna happen, I often come up with random ideas

1.0 Stargate Journey
If the mod gets to a state where practically all the important stuff from the original Stargate 1994 movie is present and all major bugs are fixed, that's when I'll start considering it a full release instead of a beta

versions beyond 1.0 would then be about adding specific stuff from Stargate SG-1, Stargate Atlantis (like the city of Atlantis) and Stargate Universe (Destiny for example) if I ever get that far

**Reply by 33mamaster** [reply]
Is there a board somewhere where we can see what you've decided to work on next and/or is ready for the next version? I'd love to follow along the passion project more closely.
Not to set in stone or formalise anything but something as simple as grouping future ideas so they're visible, or even "In progress" labelling to show an idea is coming soon.

Additionally, is there any way for other devs to help develop the mod?
I could see myself slowly contributing in my spare time if there was obvious ways I could help, and I'm sure there's others like me.
Completely understand if the decision is to keep it sole-developed.

**Reply by Povstalec** [reply]
Oh yeah, I completely forgot to reply to this, well then...

There is the Trello board that I haven't updated in a year or more, plus I kinda just jump around with ideas based on whatever I feel like doing (like for example I recently jumped from programming Jaffa AI straight to a custom cable system), so that won't really be useful here. I do post dev stuff on Discord though.

As for helping with mod development, anyone is welcome to contribute on GitHub, as long as it doesn't clash with my general vision for the mod, which just means asking me if adding/changing something is fine (Rule#1 Stargates should not have a GUI you can open).
